### PR TITLE
Remove hotspots

### PR DESCRIPTION
**Pull request by vkhramtsov**
Try to remove some hotspots. WIP.

### CODE REVIEW

**Comment by vkhramtsov** [comment]
@delphiki , please review

File: src/Captioning/Format/WebvttFile.php [hunk 1]
```diff
@@ -174,4 +158,33 @@ public function buildPart($_from, $_to)
...
+
+    /**
+     * @param string $start
+     * @param string $stop
+     * @param string $text
+     * @param string $settings
+     * @param string $id
+     * @return WebvttCue
+     */
+    private function createCue($start, $stop, $text, $settings, $id)
```
Comment (delphiki) [inline src/Captioning/Format/WebvttFile.php]: Why not overwriting the File::addCue() method?

**Comment by delphiki** [comment]
I've had a quick look at it, seems to be great.
Let me just have a better look at the WebVTT::parse refactoring and it'll be good to go.

File: src/Captioning/Format/WebvttFile.php (hunk 1 above)
Comment (vkhramtsov) [inline src/Captioning/Format/WebvttFile.php]: Because in this case we need to add

**Comment by vkhramtsov** [comment]
@delphiki , any news about this PR?

**Comment by delphiki** [comment]
All good, sorry for the delay, busy weeks lately. I'll try to be quicker next time.

Thanks again for everything!

**Pull request merged**